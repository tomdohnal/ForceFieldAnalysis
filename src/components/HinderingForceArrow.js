// @flow
import React from 'react';

import { type Force } from '../ducks/forces';
import { COLORS } from '../constants/styles';
import Arrow from '../containers/Arrow';

type Props = {
  force: Force,
}

const HinderingForceArrow = ({
  force,
}: Props) => (
  <Arrow
    force={force}
    color={COLORS.RED}
  />
);

export default HinderingForceArrow;
