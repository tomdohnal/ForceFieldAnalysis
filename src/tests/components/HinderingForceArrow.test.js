import React from 'react';
import { shallow } from 'enzyme';

import HinderingForceArrow from '../../components/HinderingForceArrow';

describe('HinderingForceArrow', () => {
  it('renders correctly with force prop', () => {
    const tree = shallow(<HinderingForceArrow force={5} />);

    expect(tree).toMatchSnapshot();
  });

  it('renders correctly with force and color prop', () => {
    const tree = shallow(<HinderingForceArrow force={5} color="#FFF" />);

    expect(tree).toMatchSnapshot();
  });
});
