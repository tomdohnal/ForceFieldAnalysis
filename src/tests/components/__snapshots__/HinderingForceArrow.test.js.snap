// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HinderingForceArrow renders correctly with force and color prop 1`] = `
<Connect(Arrow)
  color="#dc2f2f"
  force={5}
/>
`;

exports[`HinderingForceArrow renders correctly with force prop 1`] = `
<Connect(Arrow)
  color="#dc2f2f"
  force={5}
/>
`;
